tawazi/config.py

```diff
@@ -18,6 +20,14 @@ class Config(BaseSettings):
     # Weather to run the graphs in debug mode or not
     RUN_DEBUG_NODES: bool = False
 
+    # Behavior when an ExecNode is executed outside of a DAG
+    # Defaults to "Warning"
+    # Options are:
+    # - "Warning": print a warning
+    # - "Error": raise an error
+    # - "Ignore": do nothing
+    TAWAZI_EXECNODE_OUTSIDE_DAG_BEHAVIOR: XNOutsideDAGCall = XNOutsideDAGCall.error
```

Comment:
tbh, the env var has a rather long and unclear name, would you have any other ideas in mind ? Otherwise it works for me